src/Workleap.DomainEventPropagation.Publishing.Tests/TracingBehaviorTests.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace Workleap.DomainEventPropagation.Publishing.Tests;
 
+[Collection(XunitCollectionConstants.StaticActivitySensitive)]
```

Comment:
I don't think this is needed in this project as you use the attribute on a single class. Tests in a class are already sequential.

Creating collections are useful when you want to run sequentially tests from different classes in the same assembly.